go.mod

```diff
@@ -36,7 +37,7 @@ replace (
 require (
 	github.com/openshift/api v0.0.0-20200422081840-fdd1b0c14c88
 	github.com/openshift/client-go v0.0.0-20200422192633-6f6c07fc2a70
-	github.com/openshift/windows-machine-config-bootstrapper/tools/windows-node-installer v0.0.0-20200619015040-0288e17404c0
+	github.com/openshift/windows-machine-config-bootstrapper/tools/windows-node-installer v0.0.0-20200620102919-dd78396c3ed4
```

Comment:
FYI: This hash includes https://github.com/openshift/windows-machine-config-bootstrapper/pull/215. I thought you did not want to include that. Just calling that out. I have no problems with you doing so.

Reply:
https://github.com/openshift/windows-machine-config-operator/pull/72/files#r444469531

Reply:
Can't you `go get` to a particular commit hash?